INDEX

```diff
@@ -325,5 +325,13 @@
         "description": "DVR server for the Channels apps. Makes digital recordings of broadcast television using HDHomeRun network tuners. (Requires Subscription)",
         "primary_pkg": "",
         "official": false
+    },
+     "rtorrent-flood": {
+        "MANIFEST": "rtorrent-flood.json",
+        "name": "rtorrent-flood",
+        "icon": "https://www.trueos.org/iocage-icons/rtorrent-flood.json",
```

Comment:
I just realise that this needs to be an icon hosted somewhere at your side.

Reply:
I did a mistake. It's https://www.trueos.org/iocage-icons/rtorrent-flood.png.
I pushed it to the repository. You will not expose it ?

Reply:
yes we can do that.